src/templates/finops-hub/schema/schema_ea_normalized.json

```
@@ -0,0 +1,723 @@
+          "name": "InvoiceSectionName",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "IsAzureCreditEligible",
+          "type": "String"
+        },
+        "sink": {
+          "name": "IsAzureCreditEligible",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "location",
+          "type": "String"
+        },
+        "sink": {
+          "name": "Location",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "MeterCategory",
+          "type": "String"
+        },
+        "sink": {
+          "name": "MeterCategory",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "MeterId",
+          "type": "String"
+        },
+        "sink": {
+          "name": "MeterId",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "MeterName",
+          "type": "String"
+        },
+        "sink": {
+          "name": "MeterName",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "MeterRegion",
+          "type": "String"
+        },
+        "sink": {
+          "name": "MeterRegion",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "MeterSubCategory",
+          "type": "String"
+        },
+        "sink": {
+          "name": "MeterSubCategory",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "paygCostInBillingCurrency",
+          "type": "Decimal"
+        },
+        "sink": {
+          "name": "PaygCostInBillingCurrency",
+          "type": "Decimal"
+        }
+      },
+      {
+        "source": {
+          "name": "paygCostInUsd",
+          "type": "Decimal"
+        },
+        "sink": {
+          "name": "PaygCostInUsd",
+          "type": "Decimal"
+        }
+      },
+      {
+        "source": {
+          "name": "PayGPrice",
+          "type": "Decimal"
+        },
+        "sink": {
+          "name": "PayGPrice",
+          "type": "Decimal"
+        }
+      },
+      {
+        "source": {
+          "name": "previousInvoiceId",
+          "type": "String"
+        },
+        "sink": {
+          "name": "PreviousInvoiceId",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "pricingCurrency",
+          "type": "String"
+        },
+        "sink": {
+          "name": "PricingCurrency",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "PricingModel",
+          "type": "String"
+        },
+        "sink": {
+          "name": "PricingModel",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "PartNumber",
+          "type": "String"
+        },
+        "sink": {
+          "name": "ProductId",
```

PartNumber and ProductId are 2 separate things. PartNumber is critical for invoice recon in EA.
```suggestion
          "name": "PartNumber",
```

---

Done